### ISSUE TITLE
Regression of motor skills

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#5678]
**Preferred term label:**
Regression of motor skills

**Synonyms**

**Definition (free text, please give PubMed ID)**

https://www.aafp.org/afp/2015/0101/p38.html

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0002376

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://pubmed.ncbi.nlm.nih.gov/28052917/
(2 of 2 patients older than 3)

**Your nano-attribution (ORCID)**
